urdfenvs/urdf_common/generic_robot.py

```diff
@@ -198,14 +209,14 @@ def disable_velocity_control(self):
         func:`~urdfenvs.urdfCommon.generic_robot.generic_rob
         ot.apply_torque_action`
         """
-        self._friction = 0.0
         for i in range(self._n):
             p.setJointMotorControl2(
                 self._robot,
                 jointIndex=self._robot_joints[i],
                 controlMode=p.VELOCITY_CONTROL,
-                force=self._friction,
+                force=self._friction if isinstance(self._friction, float) else self._friction[i],
             )
+        p.setGravity(0, 0, 0)
```

Comment:
We are sure about this then?

Reply:
Honestly, I'm not. I will use velocity mode for my project. After that, I'll have a go at it using torque mode.

Reply:
That sounds like a good solution. In my experience the torque mode was never accurate with respect to the real world, and it is easier to use a low-level PID controller for tracking velocities in a compliant way in the real world. Then, policies can be transferred realtively smoothly.

Reply:
Really good to hear it. In that case, this one is resolved too. Thanks Max!